app/Actions/Server/InstallPrerequisites.php

```
@@ -0,0 +1,57 @@
+use App\Models\Server;
+use Lorisleiva\Actions\Concerns\AsAction;
+
+class InstallPrerequisites
+{
+    use AsAction;
+
+    public string $jobQueue = 'high';
+
+    public function handle(Server $server)
+    {
+        $supported_os_type = $server->validateOS();
```

PHPMD (2.15.0)

The variable $supported_os_type is not named in camelCase. (undefined)

(CamelCaseVariableName)